src/module/cache-container/config/thread-pools/ThreadPoolsCtrl.ts

```diff
@@ -29,7 +29,7 @@ export class ThreadPoolsCtrl extends AbstractConfigurationCtrl {
     this.containerConfigService.saveThreadPools(this.container, this.threadPools)
       .then(() => {
         if (this.isRestartRequired()) {
-          if (this.launchType.isStandaloneLocalMode()) {
+          if (this.launchType.isStandaloneMode()) {
```

Comment:
@ryanemerson having just if else switch here is ok because we want to have one type of dialog for both standalone types (clustered and non-clustered) and another for domain mode!